web/components/home/Home.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { TextInput } from '../input/text';
+import { PrimaryButton } from '../input/buttons';
+
+import sun from '../../assets/sun.svg';
+
+import skyline from '../../assets/skyline.svg';
+
+import panelBear from '../../assets/panel-bear.svg';
+import panelTrain from '../../assets/panel-train.svg';
+
+import bearCircle from '../../assets/bear-circle.svg';
+
+const getImageDiv = (alt, src, scale) => (
+  <img
+    className={scale ? 'scale-width-phablet' : ''}
+    alt={alt}
+    src={src} />
+);
+
+const rowStyle = `
+  flex jc-between ai-center dir-row
+  pad-sides-ultra pad-bottom-giga scale-pad-sides-tablet`;
+
+const Home = () => (
+  <div className="homepage overflow-hidden">
+    <div className="flex jc-between ai-center pad-top-ultra pad-sides-ultra scale-pad-sides-tablet homepage-scale-top">
+      <div>
+        <h1 className="scale-h1-tablet">nwHacks 2019</h1>
+        <p className="primary">
+          Western Canada&apos;s largest hackathon<br />
+          January 26-27, 2019 @ the University of British Columbia
+        </p>
+        <p className="secondary">
+          Get notified when registration opens!
+        </p>
+        <div className="homepage-email-registration flex ai-end">
+          <TextInput
+            placeholder="hacker@email.com" />
+          &nbsp;
+          <PrimaryButton
+            text="Submit" />
+        </div>
+        <p className="secondary">
+          <Link to="/">
+            Interested in sponsoring?
+          </Link>
+        </p>
+      </div>
+      <div className="pad-sides-xxl scale-hide-laptop">
+        {getImageDiv('sun', sun)}
+      </div>
+    </div>
+
+    <div className="homepage-skyline fill-width">
+      {getImageDiv('skyline', skyline)}
+    </div>
+
+    <div className={`${rowStyle} scale-row-desktop`}>
+      <div>
```

Comment:
Can we make each of these take 50% of the width? Kinda looks weird in 4k, the text occupies a lot more space than the image and it looks really thinned out

![homepage-4k](https://user-images.githubusercontent.com/21287547/44304118-9a158400-a307-11e8-94f0-fd918cfb25f6.png)

Reply:
saw you other comment before this but how do you define 4k and whatnot

but yes ill fix this good catch

Reply:
@mingyokim actually it might be best to leave this as is - if we scale the image up to 50% of the screen, we get massive images (to preserve aspect ratio) and tiny text

Reply:
I'm improved this a little with 1860d39

It doesnt look bad, but im not sure whats causing the borders to kinda not respect the parent div:

<img width="678" alt="image" src="https://user-images.githubusercontent.com/23356519/44385776-4c4e7680-a4d5-11e8-9a9b-d57eb02fc0d9.png">

i hate css